index.html

```diff
@@ -1,163 +1,274 @@
+            </li>
+            <li>
+              <a href="https://github.com/mpa-mxiang"
+                ><img src="./img/git.png" alt="github icon"
+              /></a>
+            </li>
+            <li>
+              <a href="#"><img src="./img/hand.png" alt="hand icon" /></a>
+            </li>
+          </ul>
+          <a><button type="button">Get my resume</button></a>
+
+          <div class="languages">
+            <h3>Languages</h3>
+            <i class="fa-solid fa-chevron-down fa-xl"></i>
+          </div>
+          <div class="language">
+            <img
+              class="language-img"
+              src="./img/js.png"
+              alt="JavaScript image"
+            />
+            <p>JavaScript</p>
+          </div>
+          <div class="language">
+            <img
+              class="language-img"
+              src="./img/html.jpg"
+              alt="JavaScript image"
+            />
+            <p>HTML</p>
+          </div>
+          <div class="language">
+            <img
+              class="language-img"
+              src="./img/css.jpg"
+              alt="JavaScript image"
+            />
+            <p>CSS</p>
+          </div>
+          <div class="languages">
+            <h3>Frameworks</h3>
+            <i class="fa-solid fa-chevron-right fa-xl"></i>
+          </div>
+          <div class="languages">
+            <h3>Skills</h3>
+            <i class="fa-solid fa-chevron-right fa-xl"></i>
+          </div>
         </div>
-        <div class="card">
-            <img class="card-image" src="./img/portfolio2.jpg" alt="webpage image">
-            <h2>Tonic</h2>
-            <div class="tags">
-                <p>CANOPY</p>
-                <img class="bullet" src="./img/Counter.png" alt="bullet">
-                <span>Back End Dev</span>
-                <img class="bullet" src="./img/Counter.png" alt="bullet">
-                <span>2015</span>
-            </div>
-            <p class="description">
-                A daily selection of privately personalized reads. No accounts or 
-                sign-ups required.
+      </div>
+      </section>
+      <section class="contact">
+        <div class="contact-div" >
+          <div class="div-content">
+          <div class="contact-h1">
+            <h1>Contact me</h1>
+          </div>
+          <div class="contact-text">
+            <p>
+              If you have an application you are interested in developing, a
+              feature that you need built or a project that needs coding. I'd
+              love to help with it.
             </p>
-            <ul class="technologies">
-                <li>html</li>
-                <li>css</li>
-                <li>javaScript</li>
-            </ul>
+          </div>
+
+          <form
+            class="form"
+            action="https://formspree.io/f/xeqwolgk"
+            method="post"
+          >
+          <div class="form-fieldset">
```

Comment:
- [ ] As rightly pointed out in the previous review, the margin between the paragraph text and the first form field is still attached. Kindly add the gap space according to the Figma template given. Once again, you'll agree with me that it does not give a great user view, but we can fix it and you get to move to the next project. Good job 👏 

# Your Design:
![image](https://user-images.githubusercontent.com/50990925/213353843-9c6699ef-46d3-4a4d-8058-ce9875e51894.png)

# Figma:
![image](https://user-images.githubusercontent.com/50990925/213353960-4552cd1f-dedf-4941-bb2d-bfd933b8b0e3.png)

Reply:
Hi,

I added padding with a new div for form-fieldset, and it looks fine when I check on my live server. 
<img width="1440" alt="Screen Shot 2023-01-19 at 12 02 20 AM" src="https://user-images.githubusercontent.com/18576811/213359753-6dfe2a21-9b31-4605-8140-d9fb0d0bc2d8.png">


I am wondering if I need to add margin between the paragraph text the first form field to have more space between them.